### DISCUSSION TITLE
Missing points in stream

### DISCUSSION THREAD

**Discussion by stigbarrett** [hozn/stravalib#472] [Q&A] [answered]
I am trying to figure out why "client.get_activity_streams" dont return the original number of points. In the test I have run, there are missing from 50 to 70 seconds in the start of the stream and the same in the end (the Garmin GPS watch have been writing per second).
anybody exprienced the same?? Anybody know what I have to du to get the full path? I have been using borh resuluyion "high" and no resulution. Same result. The tracks is about 3500 seconds.

**Comment by jsamoocha** [comment, marked as answer]
Hi! I'm quite certain this is due to a permissions issue. Strava normally hides the beginning and end of activities to the public based on a user's privacy settings. To be able to "see" these parts of an activity as a third party, the user needs to give your app permission for the `activity:read_all` scope. Can you check if that permission was granted by the user?

**Reply by stigbarrett** [reply]
I have overlooked the Scope in the autorization process. It work now. Thank you so much.
